changelog/unreleased/issue-20448.toml

```diff
@@ -0,0 +1,5 @@
+type="f"
+message="server.conf instead of graylog.conf"
```

Comment:
Please make this a sentence that users can understand when reading the changelog. Thanks!